src/pages/clusters/Pods/util.tsx

```diff
@@ -338,16 +339,23 @@ export const refreshPodsInfo = (data?: CLUSTER.ResourceTree) => {
           containerStatuses, phase, reason, message,
         } = status;
 
+        let readyCount = 0;
         let restartCount = 0;
         let onlineStatus = offlineState;
         if (containerStatuses && containerStatuses.length > 0) {
           restartCount = containerStatuses[0].restartCount;
           if (containerStatuses.length === containers.length) {
             onlineStatus = onlineState;
+            if (containers.map((c) => c.readinessProbe)
+              .filter((p) => p !== undefined && p !== null).length < containers.length) {
+              onlineStatus = unknownState;
+            }
             containerStatuses.forEach(
               (containerStatus: any) => {
                 if (!containerStatus.ready) {
                   onlineStatus = offlineState;
+                } else {
+                  readyCount += 1;
```

Comment:
For me, i would use a flag called `ready` which has a default value: `true`, then judge if any container is not ready, set `ready` to false if any exist.